### PR TITLE
Layout TBD and SignUp block(Desktop and mobile). All styles rebase in…

### PR DESCRIPTION
**Pull request by oiskritsky**
Layout TBD and SignUp block(Desktop and mobile). All styles rebase in scss.

### CODE REVIEW

File: Triphouse-project/css/Sign_up.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
+@import "../css/_variables.scss";
```
Comment (ghost) [inline Triphouse-project/css/Sign_up.scss]: 2021-04-29_22h52_16

2021-04-29_22h51_02
2021-04-29_22h51_12
style.css doen't exist on your project folder.

Comment (ghost) [reply Triphouse-project/css/Sign_up.scss]: And create folder `styles`, add to `styles` 2 folder `css` and `scss`
example of converting: sass --watch styles/scss/main.scss styles/css/style.css
 `styles/css/style.css` add to index.html

Comment (ghost) [reply Triphouse-project/css/Sign_up.scss]: Add you can import reset.scss to main.css

Comment (oiskritsky) [reply Triphouse-project/css/Sign_up.scss]: import reset.scss to main.scss (@import)

And create folder styles with to 2 folder: css and scss

**Pull request merged**